hack/update-rhcos-bootimage.py

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env python3
+# As of 4.8 we are aiming to switch to stream metadata:
```

Comment:
This is not a great place for this documentation, given that this python script is no longer used. Can we add this to a new file under docs/dev instead?

Reply:
The script unfortunately may still be used until we can port CI and other users off `rhcos.json`.  So I was planning to keep the script until that happens, but we can migrate this stuff then to docs/dev after that?  Or I guess I could make the doc now and point from there to here too.

Reply:
Oh, I missed that the rest of this file was left untouched, since the file had been deleted in the last revision that I reviewed.

Reply:
OK added a new docs/dev/pinned-coreos.md!